source/conf.py

```diff
@@ -30,6 +30,7 @@
 version = os.getenv('GITHUB_SHA')
 if not version:
     version = subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode()
+version = version[0:12]
```

Comment:
I have not observed any breaking without this patch, although I think this change is useful to de-clutter the output.

(Maybe the breaking is because you have a virus that tries to change hash values. :astonished: )

Reply:
it happens in man page rendition only.